src/openloco/input/mouse_input.cpp

```diff
@@ -1261,8 +1262,17 @@ namespace openloco::input
         switch (scrollArea)
         {
             case ui::scrollview::scroll_part::hscrollbar_button_left:
-                call(0x4c894f, regs);
+            {
+                ui::window* w = WindowManager::find((WindowType)_pressedWindowType, _pressedWindowNumber);
```

Comment:
_pressedWindowType is already a `WindowType` unsure why this needs the cast.

Reply:
Might just require an explicit deref (*)?

Reply:
You are right, it doesn't need the cast, I removed it.